### PR TITLE
Try to fix XmlTvListingsProvider

### LINKED ISSUE #1525: You can not get EPG from an xml
**Describe the bug**
Since version 10.3.6, EPG can not be obtained using an xml.

**Logs**
```
[2019-07-07 21:36:21.314 +02:00] [INF] xmltv path: "https://raw.githubusercontent.com/HelmerLuzo/TDTChannels_EPG/master/TDTChannels_EPG.xml"
[2019-07-07 21:36:21.315 +02:00] [ERR] Error adding metadata
System.Xml.XmlException: '', hexadecimal value 0x1F, is an invalid character. Line 1, position 1.
   at System.Xml.XmlTextReaderImpl.Throw(Exception e)
   at System.Xml.XmlTextReaderImpl.Throw(String res, String[] args)
   at System.Xml.XmlTextReaderImpl.ParseRootLevelWhitespace()
   at System.Xml.XmlTextReaderImpl.ParseDocumentContent()
   at System.Xml.XmlTextReaderImpl.Read()
   at System.Xml.XmlReader.ReadToDescendant(String name)
   at Emby.XmlTv.Classes.XmlTvReader.GetChannels()
   at Emby.Server.Implementations.LiveTv.Listings.XmlTvListingsProvider.GetChannels(ListingsProviderInfo info, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.GetEpgChannels(IListingsProvider provider, ListingsProviderInfo info, Boolean enableCache, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.AddMetadata(IListingsProvider provider, ListingsProviderInfo info, List`1 tunerChannels, Boolean enableCache, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.GetChannelsAsync(Boolean enableCache, CancellationToken cancellationToken)
```

**System (please complete the following information):**
 - OS: Debian Buster
 - Browser: Firefox
 - Jellyfin Version: 10.3.6
 - Reverse proxy: No

### PR DESCRIPTION
**Pull request by Bond-009**
**Changes**
Properly decompress GZ response stream.

**Issues**
Fixes #1525

### CODE REVIEW

**Comment by fruhnow** [comment]
can confirm that this fixes Issue #1525

File: Emby.Server.Implementations/LiveTv/Listings/XmlTvListingsProvider.cs [hunk 1]
```diff
@@ -66,24 +68,38 @@ private async Task<string> GetXml(string path, CancellationToken cancellationTok
...
+                new HttpRequestOptions
+                {
+                    CancellationToken = cancellationToken,
+                    Url = path,
+                    Progress = new SimpleProgress<double>(),
+                    // It's going to come back gzipped regardless of this value
+                    // So we need to make sure the decompression method is set to gzip
+                    DecompressionMethod = CompressionMethod.Gzip,
+
+                    UserAgent = "Emby/3.0"
```
Comment (DrPandemic) [inline Emby.Server.Implementations/LiveTv/Listings/XmlTvListingsProvider.cs]: Is there a reason to stick to emby here?

Comment (Bond-009) [reply Emby.Server.Implementations/LiveTv/Listings/XmlTvListingsProvider.cs]: Removed it

**Comment by joshuaboniface** [comment]
Closing as this was superseded by #1538.

**Pull request closed without merging**